test-app/ember-cli-build.js

```diff
@@ -1,11 +1,16 @@
 'use strict';
 
+const path = require('path');
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 const packageJson = require('./package');
 
 module.exports = function (defaults) {
+  const sideWatch = require('@embroider/broccoli-side-watch');
   let app = new EmberApp(defaults, {
+    trees: {
+      app: sideWatch('app', { watching: [path.join(__dirname, '../ember-resources')] }),
```

Comment:
allows me to not have to restart the app when I re-compile the addon (which is an injected dependency)